client/src/components/annotators/ImageAnnotator.vue

```diff
@@ -81,7 +81,12 @@ export default Vue.extend({
       this.frame = frame;
       this.syncedFrame = frame;
       this.emitFrame();
-      this.cacheImages();
+      const imgs = await this.cacheImages();
+      // reset dimensions of map from new image
+      const img = await imgs[this.frame];
```

Comment:
My fault, I probably left in the `async seek()` during initial testing.  I'm wondering if we really want to wait for the image to load for each seek.  During testing using a large number of different sized images scrubbing using seek would stutter.  This is different than waiting for the last image to load, it would get to the final position then load the last 3-4 images quickly and then the final image.  I believe`cacheImages()` has a thing in it to make sure it loaded the current frame before attempting to cache anything else (call to `loadFrame`).  Then if the user seeked before that first frame was loaded it would invalidate the loading.

I have an idea where it may work better if within seek we only draw the frame if the image is already loaded (checking `this.imgs[this.frame].cached or .complete`).  Then in the `onload` for images we check to see if the currently completed `this.frame` is equal the `onload` frame and if it is we attempt to do the resize and drawing.  I think this should improve some of the seeking a bit as well as handle the different image sizes.  I realize that it may warrant some more discussion.

Below is a brief clip of the PR's behavior.  Note there are errors in the console as well because when seeking the current img may not be fully loaded causing it to try to reset using 0 width and height causing an error in `geoViewer.bounds` function.
![imagecacheResize](https://user-images.githubusercontent.com/61746913/100746426-b04b8f00-33ae-11eb-881f-021b726aa731.gif)